rpcserver.go

```diff
@@ -637,6 +637,70 @@ func createVinList(mtx *wire.MsgTx) []btcjson.Vin {
 	return vinList
 }
 
+// createVinList returns a slice of JSON objects for the inputs of the passed
+// transaction.
+func createVinListPrevOut(s *rpcServer, mtx *wire.MsgTx, chainParams *chaincfg.Params, vinExtra int) []btcjson.VinPrevOut {
+	vinList := make([]btcjson.VinPrevOut, len(mtx.TxIn))
+
+	var txStore blockchain.TxStore
+	if vinExtra == 1 {
```

Comment:
On second thought, this (and the other one below) should be `!= 0`.  0 is false, non-zero is true.  With this check specifying 2 would be treated as false which wouldn't make much sense.

That said, I actually would rather the parameter be a boolean.  I know the existing `Verbose` field is an `int`, but that is only because it is intentionally being compatible with the Bitcoin Core `searchrawtranscations` RPC call which for some reason uses an integer instead of a JSON bool.  Since this is extended the in a custom way for btcd, I'd rather use the right type for the job.